pkg/new-ui/v1beta1/frontend/src/polyfills.ts

```diff
@@ -0,0 +1,62 @@
+/**
```

Comment:
This was introduced because I initialized the project with Angular's CLI.
During the built process this file gets filled with necessary import statements etc. This file is always produced from the built process and also gets requested from index.html

https://angular.io/guide/browser-support#enabling-polyfills-with-cli-projects

Reply:
Got it, thanks!